docs/guides/1.4.0-upgrade-guide.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: "couchbase-capella"
+page_title: "Couchbase Capella Provider 1.4.0: Upgrade and Information Guide"
+sidebar_current: "docs-couchbase-capella-guides-140-upgrade-guide"
+description: |-
+Couchbase Capella Provider 1.4.0: Upgrade and Information Guide
+---
+
+
+# Couchbase Capella Provider 1.4.0: Upgrade and Information Guide
+
+New features in 1.4.0
+
+
+## New Features
+* Enable Flush Bucket [`couchbase-capella_flush`](https://registry.terraform.io/providers/couchbasecloud/couchbase-capella/latest/docs/resources/flush)
+* Enable Azure Network Peer [`couchbase-capella_network_peer`](https://registry.terraform.io/providers/couchbasecloud/couchbase-capella/latest/docs/resources/network_peer)
+
+
+## Flush Bucket
+Use the new `flush` resource to flush a bucket
+```
+output "new_bucket" {
+    value = couchbase-capella_bucket.new_bucket
+}
+```
+```
+output "bucket_id" {
+    value = couchbase-capella_bucket.new_bucket.id
+}
+```
+```
+resource "couchbase-capella_bucket" "new_bucket" {
+    name                       = var.bucket.name
+    organization_id            = var.organization_id
+    project_id                 = var.project_id
+    cluster_id                 = var.cluster_id
+    type                       = var.bucket.type
+    storage_backend            = var.bucket.storage_backend
+    memory_allocation_in_mb    = var.bucket.memory_allocation_in_mb
+    bucket_conflict_resolution = var.bucket.bucket_conflict_resolution
+    durability_level           = var.bucket.durability_level
+    replicas                   = var.bucket.replicas
+    flush                      = var.bucket.flush
+    time_to_live_in_seconds    = var.bucket.time_to_live_in_seconds
+    eviction_policy            = var.bucket.eviction_policy
+}
+```
+```
+output "new_flush" {
+    value = couchbase-capella_flush.new_flush
+}
+```
+```
+resource "couchbase-capella_flush" "new_flush" {
+    organization_id = var.organization_id
+    project_id      = var.project_id
+    cluster_id      = var.cluster_id
+    bucket_id       = var.bucket_id
+}
+```
+For more information, see the [flush bucket examples](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/tree/main/examples/flush_bucket).
+
+## Azure Network Peer
+
+Use the `azure_network_peer_command` data source to get an AWS command for configuring VPC endpoint.
```

Comment:
```suggestion
Use the `azure_network_peer_command` data source to get an Azure CLI command for configuring VNET endpoint.
```

General: I have not paid close attention but please double check to ensure no copy paste errors. @PaulomeeCb  @a-atri